### DISCUSSION TITLE
Kicks is a continuation of @jondot's Sneakers

### DISCUSSION THREAD

**Discussion by michaelklishin** [ruby-amqp/kicks#1] [Announcements]
Kicks is a high-performance RabbitMQ background processing framework for
Ruby, originally developed by @jondot 👏 under the name of [Sneakers](https://github.com/jondot/sneakers).
The original repo was abandoned by the person who has exclusive control
over the RubyGems project, so the community of users has decided to move it
to this "fork" (continuation) under the new name.

By virtue of its Sneakers lineage, Kicks is a mature project that has been around since 2016.

## Short Term Plan

We can do the following to proceed, since the MIT license permits us to do so:

* [x] Push this repo as is there
* [x] Update the README to credit @jondot for the original Sneakers project
* [ ] Ask authors of open PRs to re-submit them there
* [x] Bump the major version to indicate the shift to the new repo with a new set of maintainers. 
* [x] Update this repo's README to reflect the move to the new home
* [ ] Release the current state of the project **without renaming anything** as `kicks` X.0
* [ ] If we choose to — and I don't think we have to or should — we can rename the modules and so on to `Kicks`

## Come Join the Team of Kicks Maintainers

I do not volunteer to be a maintainer but since I work on RabbitMQ and (as time permits) Bunny, you can be sure that I at least will be around to grant permissions to those willing to maintain Kicks.

I will also be around to not allow for any decisions that Team RabbitMQ would find questionable in such a library/tool. Or at least serve as a "litmus test" for ideas and changes.

If you want to be on the Kicks maintainers team (for real, maintaining OSS software can be nerve wrecking and time consuming), just [start a Discussion](https://github.com/ruby-amqp/kicks/discussions) in this repo.

**Comment by remmons** [comment]
@michaelklishin do you have sense for when you'll cut the first release for kicks?

**Comment by michaelklishin** [comment]
@remmons honestly, we can do that any day. @simi WDYT?

We can ship `3.0.0` to let folks switch over from Sneakers and then https://github.com/ruby-amqp/kicks/pull/12 can go into a `3.1.0` release.

**Reply by michaelklishin** [reply]
@simi any objections to cutting 3.0 to get the ball rolling, and then including #12 into `3.1.0`?

**Reply by remmons** [reply]
Following up here. Any chance the first release can be cut now that #12 has been merged, @michaelklishin, @simi?

**Reply by simi** [reply]
Happy with new release, yes.

**Reply by michaelklishin** [reply]
I can cut a new release some time this week.
